doc/validation/diffraction.rst

```diff
@@ -252,6 +255,124 @@ and :math:`\boldsymbol{e}_N = \boldsymbol{e}_r(0) \times \boldsymbol{e}_E`.
 
 .. [1] https://calgary.rasc.ca/field_rotation.htm
 
+
+FFT Mode
+^^^^^^^^
+
+Conceptionally, to model diffraction in FFT mode, the image is
+convoluted with the PSF of the diffraction effect. This is implemented
+by rendering the PSF to a finite square and convolve the square which each
```

Comment:
for a single pixel and then convolving that pattern with a subset of the pixels in the image.

Reply:
Changed to 

> This is implemented by convolving the pattern for a single pixel with a subset of
> the pixels in the image.
> For each object a region of saturated pixels ...

But is not now the 2nd sentence a bit redundant?